src/main/java/duke/exception/BobiException.java

```diff
@@ -0,0 +1,19 @@
+package duke.exception;
```

Comment:
I like how you really considered alot of exceptions!

Reply:
thank you!